r/tests/testthat/test-dplyr-string-functions.R

```diff
@@ -719,6 +719,55 @@ test_that("errors in strptime", {
   )
 })
 
+test_that("strftime", {
+  # TODO: consider reevaluating this workaround after ARROW-12980
+  withr::local_timezone("UTC")
+  t_stamp <- tibble(x = c(lubridate::ymd_hms("2018-10-07 19:04:05"), NA))
+  t_string <- tibble(x = c("2018-10-07 19:04:05.000000", NA))
+  t_string_2 <- tibble(x = c("07 October 2018 19:04:05.000000 UTC", NA))
+
+  # TODO: remove once we have tz support on windows ARROW-13168
+  if (tolower(Sys.info()[["sysname"]]) != "windows") {
+    expect_equal(
```

Comment:
We've got a custom function `expect_dplyr_equal()`, which we tend to use with a lot of these functions - they make it easier to ensure that the behaviour of the implemented Arrow function matches that of the equivalent base R or tidyverse function.  Would you mind updating these tests to use that helper function?

Reply:
Good point! switched to `skip_on_os("windows")`.